server/controllers/group_controller.js

```diff
@@ -1,37 +1,37 @@
 import Group from '../models/group';
 
-const ObjectId = require('mongoose').Types.ObjectId;
-
-export const createGroup = (name, description, creator) => {
-  // TODO: error check on names, descriptions and creators?
+export const createGroup = (name, description, userId) => {
   const group = new Group();
   group.name = name;
   group.description = description;
-  group.creator = creator; // TODO: @ploomis how to get authenticated user ??
+  group.creator = userId; // how to get authenticated user ??
+
   group.createDate = Date.now();
   group.editDate = Date.now();
-  group.members.push(ObjectId(creator));   // TODO: make sure creator is a valid user ID
+  group.members.push(userId);   // TODO: make sure creator is a valid user ID
 
   return group.save(); // TODO: catch errors
 };
 
-export const addGroupMember = (groupid, memberid) => {
+export const addGroupMember = (groupIds, userId) => {
```

Comment:
Maybe... I was thinking about making it like addMemberToGroups since it's a single user and multiple groups (I feel like addGroupMembers sounds like multiple members to one group). @charblanc any thoughts?